### DISCUSSION TITLE
Pimcore Website on front-end is not found  back-end (how to edit)

### DISCUSSION THREAD

**Discussion by ismetsener** [pimcore/pimcore#8169] [Q&A / Support]
Hi,

In Pimcore Documents I can see all the web pages.
Here I can edit the webpages etc

But Some webpages I cannot see why?
for example:
front site:in the menu find&order : 
Popular Brands
On sale
Cars
Spare Parts

but in the admin panel I see in find&order:
Popular Brands
On sale

So Cars and Spare Parts is not shown

**Comment by ismetsener** [comment]
I should see  all the pages of the front -end    also on the backend??

**Comment by dvesh3** [comment]
@ismetsener these are [custom routes](https://pimcore.com/docs/pimcore/current/Development_Documentation/MVC/Routing_and_URLs/index.html#page_5-Static-Routes-Custom-Routes) which are generated on fly and does not require any editing from backend. 

If you wonder, how these are generated in Navigation: we have a Twig extension for Navigation links generation [app_navigation_data_links](https://github.com/pimcore/demo/blob/4eee715d10a4ffc5500e3da5ca843d9e270ed775/src/AppBundle/Twig/Extension/NavigationExtension.php#L74) called on [layout](https://github.com/pimcore/demo/blob/ffc5837f555ebe6e024fafc1d8fe9d743d8ecd8d/app/Resources/views/layouts/layout.html.twig#L85), which checks if a document has "root_category" property, 
![image](https://user-images.githubusercontent.com/5137917/108480226-f8d06580-7296-11eb-8996-eb2ce78c2a36.png)

if exists then it creates the structure from object tree with url pattern defined [here](https://github.com/pimcore/demo/blob/96ff2bca03aa4189b75a0858b7e1a084bdb28721/src/AppBundle/Controller/ProductController.php#L58).

more info about Custom Routes: https://pimcore.com/docs/pimcore/current/Development_Documentation/MVC/Routing_and_URLs/Custom_Routes.html

**Comment by ismetsener** [comment]
suppose that you want to edit,  how is this possible?
which steps?
